dissect/disc/c_iso_9660.py

```diff
@@ -0,0 +1,135 @@
+from dissect.cstruct import cstruct
+
+ISO_DEF = """
+
+/* volume descriptor types */
```

Comment:
```suggestion
iso_def = """
/* Volume descriptor types */
```
Not bad per se, but more consistent with other projects.